src/index.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Routes from './Routes';
+import '../src/Styles/common.scss';
+import '../src/Styles/reset.scss';
```

Comment:
```suggestion
import './Styles/common.scss';
import './Styles/reset.scss';
```
의미는 같지만 짧은 방법으로 경로 수정해주세요.